### ISSUE TITLE
Warning: Cannot update a component ($View) while rendering a different component

### ISSUE THREAD

**Issue by jmeistrich** [LegendApp/legend-list#49]
FYI for anyone who sees this warning: it's not really a problem so you can ignore it. I had temporarily disabled it while developing so didn't notice it, but I will fix it soon so the warning stops happening.

**Comment by todorone** [comment]
I see other issue for some of lists, is it related or it`s another issue?
```
(NOBRIDGE) ERROR  Warning: Cannot update a component (`Container`) while rendering a different component (`ForwardRef(LegendListInner2)`). To locate the bad setState() call inside `ForwardRef(LegendListInner2)`, follow the stack trace as described in https://react.dev/link/setstate-in-render
    in LegendListInner2
    in StateProvider
    in LegendList2 (created by PickMediaScreen)
```

**Comment by jmeistrich** [comment]
It's the same issue.

**Comment by jmeistrich** [comment]
(Note mostly to self)

A lot of this was resolved by #43 but I'm still seeing it happening when testing in the Bluesky app

```
Cannot update a component (`Container`) while rendering a different component (`ForwardRef(LegendListInner2)`)
```

**Comment by jmeistrich** [comment]
I think this should be fixed after #58 in version 0.6.3. Is anybody still seeing it? If so can you share the text of the error and what appears to cause it?

**Comment by todorone** [comment]
@jmeistrich Can confirm, it has gone in my case. Thank you for amazing work! 💙
